src/runtime/types.cpp

```diff
@@ -3728,11 +3728,12 @@ void setupRuntime() {
         BoxedClass(object_cls, BoxedCApiFunction::gcHandler, 0, 0, sizeof(BoxedCApiFunction), false, "capifunc");
     method_cls = new (0) BoxedClass(object_cls, BoxedMethodDescriptor::gcHandler, 0, 0, sizeof(BoxedMethodDescriptor),
                                     false, "method_descriptor");
+    method_cls->giveAttr("__name__", boxString("method_descriptor"));
     wrapperobject_cls = new (0) BoxedClass(object_cls, BoxedWrapperObject::gcHandler, 0, 0, sizeof(BoxedWrapperObject),
                                            false, "method-wrapper");
     wrapperdescr_cls = new (0) BoxedClass(object_cls, BoxedWrapperDescriptor::gcHandler, 0, 0,
                                           sizeof(BoxedWrapperDescriptor), false, "wrapper_descriptor");
-
+    wrapperdescr_cls->giveAttr("__name__", boxString("wrapper_descriptor"));
```

Comment:
Same comment for this one -- `print int.__add__.__name__` should give `'__add__'`.